--- core/restservices.class.inc.php	2017-05-24 19:10:31.156456256 +0800
+++ /tmp/restservices.class.inc.php	2017-05-24 19:10:46.933503094 +0800
@@ -102,6 +102,10 @@
 					$value[] = $aLnkValues;
 				}
 			}
+			elseif($oAttDef instanceof AttributeCustomFields)
+			{
+				$value = $oObject->Get($sAttCode)->GetValues();
+			}
 			else
 			{
 				$value = $oAttDef->GetForJSON($oObject->Get($sAttCode));
